src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,20 +243,58 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
 
             private void AddExceptions(ISymbol symbol)
             {
-                var exceptionTypes = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true).ExceptionTypes;
+                const int maxTextSize = 100;
+                var documentationComment = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true);
+                var exceptionTypes = documentationComment.ExceptionTypes;
+                var formatter = Services.GetLanguageServices(_semanticModel.Language).GetRequiredService<IDocumentationCommentFormattingService>();
                 if (exceptionTypes.Any())
                 {
-                    var parts = new List<SymbolDisplayPart>();
+                    var parts = new List<TaggedText>();
                     parts.AddLineBreak();
                     parts.AddText(WorkspacesResources.Exceptions_colon);
+                    var textSize = 0;
                     foreach (var exceptionString in exceptionTypes)
                     {
-                        parts.AddRange(LineBreak());
-                        parts.AddRange(Space(count: 2));
-                        parts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel));
+                        var checkedParts = new List<TaggedText>();
+                        checkedParts.AddRange(LineBreak().ToTaggedText());
+                        checkedParts.AddRange(Space(count: 2).ToTaggedText());
+                        checkedParts.AddRange(AbstractDocumentationCommentFormattingService.CrefToSymbolDisplayParts(exceptionString, _position, _semanticModel).ToTaggedText());
+                        var exceptionTexts = documentationComment.GetExceptionTexts(exceptionString);
+                        if (exceptionTexts.Length > 0)
```

Comment:
Can there ever be more than 1 of these?

Reply:
I think I did that in case there was for some reason a chance that ``GetExceptionTexts`` returns an empty Enumerable.